scout/server/blueprints/variant/templates/variant/sv-variant.html

```diff
@@ -37,307 +38,174 @@
 
 {% block content_main %}
   {% if variant.missing_data %}
     <div class="alert alert-warning">Showing only first 30 genes!</div>
   {% endif %}
-  <div class="row">
+    <div class="row">
+      <div class="col-md-6">
+        <div class="card panel-default">
+          <div class="panel-heading">Basics</div>
+            <div class="card-body">
+            <ul class="list-group">
+              <li class="list-group-item">Rank <span class="badge badge-pill badge-secondary float-right">{{ variant.variant_rank }}</span></li>
+              <li class="list-group-item">
+                Rank score
+                <span class="badge badge-pill badge-secondary float-right">{{ variant.rank_score }}</span>
+              </li>
+
+              <li class="list-group-item">
+                Category
+                <span class="badge badge-pill badge-secondary float-right">{{ variant.sub_category|upper }}</span>
+              </li>
+
+              <div class="list-group-item">
+                <strong>Gene panels:</strong><br>
+                {% for panel_id in variant.panels %}
+                  <a href="{{ url_for('panels.panel', panel_id=panel_id) }}">{{ panel_id }}</a>&nbsp;&nbsp;
+                {% endfor %}
+              </div>
+              <div class="list-group mt-3">
+                <div>
+                  <form action="{{ url_for('variant.variant_update', institute_id=institute._id, case_name=case.display_name, variant_id=variant._id) }}" method="POST">
+                    <label class="ml-3">Variant tag</label>
+                    <div class="row">
+                      <div class="col-8">
+                        <select name="manual_rank" class="form-control ml-3">
+                          <option value="-1">Select a tag...</option>
+                          {% for rank, data in manual_rank_options.items() %}
+                            <option {% if rank == variant.manual_rank %} selected {% endif %} value="{{ rank }}">
+                              {{ data.label }}
+                            </option>
+                          {% endfor %}
+                        </select>
+                      </div>
+                      <div class="col-4">
+                        <button type="submit" class="btn btn-outline-secondary form-control">Save</button>
+                      </div>
+                    </div>
+                  </form>
+                </div>
+              </div>
+              <div class="list-group mt-3">
+                <div>
+                  <form action="{{ url_for('variant.variant_update', institute_id=institute._id, case_name=case.display_name, variant_id=variant._id) }}" method="POST">
+                    <label class="ml-3">Dismiss variant</label>
+                    <div class="row">
+                      <div class="col-8 d-flex">
+                          <select multiple class="ml-3 selectpicker" title="Select categories..." name="dismiss_variant" id="dismiss_variant" >
```

Comment:
Right, this popped back again:
![Screenshot 2020-02-05 at 16 39 37](https://user-images.githubusercontent.com/758570/73856793-4a49b600-4836-11ea-8ea2-73f04de0c002.png)